packages/fiori/src/ViewSettingsDialog.ts

```diff
@@ -514,9 +534,12 @@ class ViewSettingsDialog extends UI5Element {
 		}
 
 		this.fireEvent("before-open", {}, true, false);
-		this._dialog.open = true;
+	}
+
+	afterDialogOpen(): void {
+		this._dialog?.querySelector<List>("[ui5-list]")?.focusFirstItem();
```

Comment:
Seems possible to apply focus on the first item and then apply the focus on the first recently used item.
Could we apply the focus once according to the given conditions?

Reply:
I don't think we have stable condition to base the check on. the condition used in the focus method will not work, because if the case where you select filters, then select something in the sortby list, then the focus just dissapears even tho we have an element which will make the condition true. And if we don't base it on this check, even if the check in the method is not working, the focus on the first element will be applied. As of why the check is not working I think is something that we need to look at separately.